templates/web/fixmystreet.com/report/new/extra_name.html

```diff
@@ -0,0 +1,40 @@
+[%# Same as parent, with added gender question %]
+
+[% IF c.cobrand.ask_gender_question %]
+<p>
+We’re conducting research to make sure our site is good for everyone;
+please help us out by providing your gender:
+</p>
+
+<p class="radio-segmented-control" style="font-size: 80%">
```

Comment:
This is included on the update form too, but any answer given by the user is discarded. Perhaps this shouldn't be shown on the update form?